tests/test_footprintMergeCatalog.py

```diff
@@ -102,35 +102,49 @@ def setUp(self):
         insertPsf(pos3, im3, psf3, kernelSize, flux)
 
         schema = afwTable.SourceTable.makeMinimalSchema()
-        self.idFactory = afwTable.IdFactory.makeSimple()
-        self.table = afwTable.SourceTable.make(schema, self.idFactory)
 
         # Create SourceCatalogs from these objects
         fp1 = afwDetect.FootprintSet(
             im1, afwDetect.Threshold(0.001), "DETECTED")
-        self.catalog1 = afwTable.SourceCatalog(self.table)
+        # Use a different ID factory for each footprint so that the peak ID's
+        # overlap. This allows us to test ID collisions.
+        idFactory = afwTable.IdFactory.makeSimple()
+        table = afwTable.SourceTable.make(schema, idFactory)
+        self.catalog1 = afwTable.SourceCatalog(table)
         fp1.makeSources(self.catalog1)
 
+        idFactory = afwTable.IdFactory.makeSimple()
+        table = afwTable.SourceTable.make(schema, idFactory)
```

Comment:
Since you're re-using the `idFactory` name every time here, it might be clearer to just put the `IdFactor.makeSimple()` inside the `SourceTable.make()` call in each of these cases?